packages/synapse-react-client/src/components/EntityBadgeIcons/EntityBadgeIcons.tsx

```diff
@@ -118,7 +124,25 @@ export const EntityBadgeIcons = (props: EntityBadgeIconsProps) => {
   const [schemaConformance, setSchemaConformance] = useState(
     SchemaConformanceState.NO_SCHEMA,
   )
-
+  const getAnnotationDisplayValue = (
+    value:
+      | string
+      | number
+      | boolean
+      | string[]
+      | number[]
+      | boolean[]
+      | undefined,
+  ) => {
+    if (typeof value === 'string') {
+      return value
+    } else if (Array.isArray(value)) {
+      return value.join(', ')
+    } else {
+      // Handle number and boolean cases if needed
+      return value?.toString()
+    }
+  }
```

Comment:
you may be able to reuse / modify `getDisplayedAnnotation` in AnnotationsTable.tsx

Reply:
Reusing, which improves the rendering of datetimes.  Great idea